webview-ui/src/components/cline-rules/ClineRulesToggleModal.tsx

```diff
@@ -42,13 +45,18 @@ const ClineRulesToggleModal: React.FC = () => {
 		setRemoteRulesToggles,
 		setRemoteWorkflowToggles,
 	} = useExtensionState()
+	const [globalHooks, setGlobalHooks] = useState<Array<{ name: string; enabled: boolean; absolutePath: string }>>([])
+	const [workspaceHooks, setWorkspaceHooks] = useState<
+		Array<{ workspaceName: string; hooks: Array<{ name: string; enabled: boolean; absolutePath: string }> }>
+	>([])
+	const [isWindows, setIsWindows] = useState(false)
```

Comment:
We could use the isMacOSOrLinux helper here:

```suggestion
import { isMacOSOrLinux } from "@/utils/platformUtils"

	const isWindow = !isMacOSOrLinux()
```